lib/rubocop/cop/layout/rescue_ensure_alignment.rb

```diff
@@ -29,13 +29,25 @@ class RescueEnsureAlignment < Cop
               '%<begin_loc_line>d, %<begin_loc_column>d.'.freeze
         ANCESTOR_TYPES = %i[kwbegin def defs class module].freeze
         RUBY_2_5_ANCESTOR_TYPES = (ANCESTOR_TYPES + [:block]).freeze
+        ANCESTOR_TYPES_WITH_ACCESS_MODIFIERS = %i[def defs].freeze
 
         def on_resbody(node)
-          check(node) unless modifier?(node)
+          check node unless modifier? node
```

Comment:
There is a strong preference towards using parentheses in the project.

Reply:
We would automatically check this, but the cop that does that still has too many false positives to be enabled.

But yes, please don't remove the parens for regular method calls (non DSLs) in this project. 🙇

Reply:
fixed